projects/program-website/src/app/core/services/firebase.service.ts~HEAD

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
```

Comment:
You probably added the file by accident.
Could you please remove it?

Reply:
Handled.